include/openthread/netdata.h

```diff
@@ -230,6 +230,19 @@ struct otJoinerDiscerner;
 otError otNetDataSteeringDataCheckJoinerWithDiscerner(otInstance *                    aInstance,
                                                       const struct otJoinerDiscerner *aDiscerner);
 
+/**
+ * This function checks whether the Leader's Network Data contains a given OMR prefix.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aPrefix    A pointer to the IPv6 prefix.
+ *
+ * @returns  Whether Leader's Network Data contains the OMR prefix @p aPrefix.
```

Comment:
What if `aPrefix` is not a valid OMR prefix itself (e.g., worng lengt)?  I guess we return `false` in that case (even if the netdata contains a matching prefix to `aPrefix)). Should we document the behavior here?

Reply:
done